app/src/main/java/org/akvo/flow/activity/SurveyActivity.java

```diff
@@ -455,12 +475,26 @@ private void displayForm(String surveyedLocaleId, User user) {
                 formInstanceId, readOnly, mSurveyGroup);
     }
 
-    private void displaySelectedUser() {
-        User user = FlowApp.getApp().getUser();
-        if (user != null) {
-            Toast.makeText(this, getString(R.string.logged_in_as) + " " + user.getName(),
-                    Toast.LENGTH_LONG).show();
-        }
+    private void verifySelectedUser() {
```

Comment:
I was trying to understand why you changed this method to `verifySelectedUser()` because it seems to me the main objective stays the same., i.e. display the user, but before displaying we verify that the username exists.

Reply:
Ok, you are right I will rename it back